src/server/api/controllers/users.controller.js

```diff
@@ -26,7 +26,23 @@ const getUsersById = async (id) => {
     return error.message;
   }
 };
-
+// eslint-disable-next-line
+const getUserFavorites = async (user_id) => {
+  // eslint-disable-next-line
+   if (!user_id) {
```

Comment:
Hey Bikram, This part is not working now. You can use parseFloat for it.
```suggestion
   if (!parseFloat(user_id)) {
```

Reply:
why float? since its expecting Integer it should be parseInt()  and >0 - based on error.
But i think everyone including me just copied tis code from midules and didnt do it right 😄

Reply:
Actually I used this in user patch. I dont remember but I was get an error with parseInt(). But maybe it is my mistake. Both of them will work correctly. You can use parseInt() too.